cmd/agent/common/commands/check.go

```diff
@@ -568,14 +567,7 @@ func writeCheckToFile(checkName string, checkFileOutput *bytes.Buffer) {
 	filenameSafeTimeStamp := strings.ReplaceAll(time.Now().UTC().Format(time.RFC3339), ":", "-")
 	flarePath := filepath.Join(common.DefaultCheckFlareDirectory, "check_"+checkName+"_"+filenameSafeTimeStamp+".log")
 
-	w, err := scrubber.NewWriter(flarePath, os.ModePerm)
-	if err != nil {
-		fmt.Println("Error while writing the check file:", err)
-		return
-	}
-	defer w.Close()
-
-	_, err = w.Write(checkFileOutput.Bytes())
+	err := ioutil.WriteFile(flarePath, checkFileOutput.Bytes(), os.ModePerm)
```

Comment:
You're assuming that these bytes can be written to the disk without going through the scrubbing logic, could you confirm that it's safe to do so because of their content? I think it's worth explicitly calling out in a comment as well

Reply:
Oh, good catch -- that was supposed to have a ScrubBytes call!